docs/storage-schema.md

```diff
@@ -31,6 +31,10 @@ Heapster exports the following metrics to its backends.
 | memory/cache | Cache memory usage. |
 | memory/rss | RSS memory usage. |
 | memory/working_set | Total working set usage. Working set is the memory being used and not easily dropped by the kernel. |
+| accelerator/memory_capacity | Memory capacity of an accelerator . |
+| accelerator/memory_usage | Memory used of an accelerator . |
+| accelerator/utility | Utility of an accelerator . |
```

Comment:
Use the names that are used in the upstream API.
```
accelerator/memory_total
accelerator/memory_used
accelerator/duty_cycle
```

Reply:
Will do.